docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -350,8 +350,41 @@ Excel.run(function (context) {
 
 You can also find the cell responsible for spilling into a given cell by using the [Range.getSpillParent](/javascript/api/excel/excel.range#getspillparent--) method. Note that `getSpillParent` only works when the range object is a single cell. Calling `getSpillParent` on a range with multiple cells will result in an error being thrown (or a null range being returned for `Range.getSpillParentOrNullObject`).
 
+## Trace formula precedents
+
+An Excel formula often refers to other cells or groups of cells. These cells or groups of cells are known as "precedents" to the formula. To learn more about Excel features related to relationships between cells, see the [Display the relationships between formulas and cells](https://support.microsoft.com/office/display-the-relationships-between-formulas-and-cells-a59bef2b-3701-46bf-8ff1-d3518771d507) article. 
```

Comment:
The SOC article avoids this grammatical construction (precedents to/of a formula) entirely, and my other searches indicate that *only* Microsoft uses the terms formula and precedent together. The closest aligning phrase in the SOC article is "cells that provide data to a formula (precedents)" which is how I ended up using 'to' instead of 'of'. Is there anywhere else you suggest I search? My inclination is to make sure I'm consistent but that this is a gray area, or language in development. Perhaps it's a grammatical construction to avoid entirely, but I think dancing around the language makes the concept more confusing.

Reply:
The place you hear "precedent" the most is in legal documentation. You would say "a legal precedent". Would it make sense to frame it similarly here? Like, "These cells or groups of cells are known as formula "precedents"." ?

Reply:
Yes, I agree! I will rephrase this section and others to avoid the awkward preposition.